styles/Home.module.css

```diff
@@ -1,7 +1,3 @@
-.container {
-  padding: 0 2rem;
-}
-
 .main {
```

Comment:
[curious] will you be removing/migrating the existing styling?

Reply:
This is done in #40. I wanted to just get started with Emotion and set it up in this branch and then do the heavier work of transferring everything else over separately, just in case it was more complicated than I thought.